R/init.R

```diff
@@ -1,87 +1,109 @@
-if (interactive() &&
-  Sys.getenv("RSTUDIO") == "" &&
-  Sys.getenv("TERM_PROGRAM") == "vscode") {
-  if (requireNamespace("jsonlite", quietly = TRUE)) local({
-    # cleanup previous version
-    removeTaskCallback("vscode-R")
-    options(vscodeR = NULL)
-
-    .vsc.name <- "tools:vscode"
-    if (.vsc.name %in% search()) {
-      detach(.vsc.name, character.only = TRUE)
-    }
-
-    .vsc <- local({
-      pid <- Sys.getpid()
-      wd <- getwd()
-      tempdir <- tempdir()
-      homedir <- Sys.getenv(
-        if (.Platform$OS.type == "windows") "USERPROFILE" else "HOME"
+local({
+  if (interactive() &&
+    Sys.getenv("RSTUDIO") == "" &&
+    Sys.getenv("TERM_PROGRAM") == "vscode") {
```

Comment:
Just a trival issue, hope you don't mind:  

Will rewrite this like below better? as it reduces the layer of the code and it clearly states that we only execute the code under such condition and eval in a `local()` environment.

```r
 if (interactive() &&
    Sys.getenv("RSTUDIO") == "" &&
    Sys.getenv("TERM_PROGRAM") == "vscode") local({
 # code 
 })

Reply:
Yes you are right. It looks better now.